app/views/assignments/find_client.html.slim

```diff
@@ -0,0 +1,91 @@
+.row
+  .col-xs-12
+    h1= @volunteer.contact.full_name
+
+.table-responsive
+  table.table.table-no-border-top
+    tbody
+      = render 'contacts/show', contact: @volunteer.contact
+      tr
+        td= t_attr(:date_of_birth)
+        td= l(@volunteer.date_of_birth) if @volunteer.date_of_birth
+      tr
+        td= t_attr(:gender)
+        td= t("gender.#{@volunteer.gender}") if @volunteer.gender.present?
```

Comment:
You can just do `gender?` instead of `gender.present?` (works for all model attributes, but not associations)